DeploymentCloud/Deployment.Common/Helpers/UtilityModule.psm1

```diff
@@ -564,17 +564,19 @@ function Set-AzureAADAccessControl([string]$AppId) {
     $ErrorActionPreference = "stop"
 }
 
-function Set-AzureAADApiPermission([string]$ServiceAppId, [string]$ClientAppId) {
+function Set-AzureAADApiPermission([string]$ServiceAppId, [string]$ClientAppId, [string]$RoleId) {
```

Comment:
does this change any of our admin steps docs?

Reply:
No, For admin step, we ask user to run runAdminSteps.bat and no change for it.